doc/apidoc/Hazelcast.IHazelcastClient.md

```diff
@@ -7,16 +7,12 @@ by a @Hazelcast.HazelcastClientFactory. Before it can be used, it needs to be op
 in order to properly release its resources. For example:
 
 ```csharp
-var options = HazelcastOptions.Build();
-var factory = new HazelcastClientFactory(options);
-var client = factory.CreateClient();
-await client.OpenAsync();
+var options = new HazelcastOptionsBuilder.Build();
+var client = await HazelcastClientFactory.StartNewClientAsync();
```

Comment:
Built `options` doesn't seem to be used here.

Reply:
It should, fixing.